test/functional/tests/security/test_security_user.py

```diff
@@ -0,0 +1,468 @@
+
+    with TestRun.step("Try to flush cache."):
+        try:
+            output = run_as_other_user(cli.flush_cache_cmd(str(cache.cache_id)), user_name)
+            if output.exit_code == 0:
+                TestRun.fail("Flushing cache should fail!")
+        except CmdException:
+            TestRun.LOGGER.info("Non-root user cannot flush cache.")
+
+    with TestRun.step("Try to flush core."):
+        try:
+            output = run_as_other_user(cli.flush_core_cmd(str(cache.cache_id),
+                                                          str(core.core_id)), user_name)
+            if output.exit_code == 0:
+                TestRun.fail("Flushing core should fail!")
+        except CmdException:
+            TestRun.LOGGER.info("Non-root user cannot flush core.")
+
+    with TestRun.step("Try to set cleaning policy and its parameters."):
+        try:
+            output = run_as_other_user(cli.set_param_cleaning_cmd(
+                str(cache.cache_id), "nop"), user_name)
+            if output.exit_code == 0:
+                TestRun.fail("Setting cleaning policy should fail!")
+            output = run_as_other_user(cli.set_param_cleaning_cmd(
+                str(cache.cache_id), "alru"), user_name)
+            if output.exit_code == 0:
+                TestRun.fail("Setting cleaning policy should fail!")
+            try:
+                output = run_as_other_user(cli.set_param_cleaning_alru_cmd(str(cache.cache_id),
+                                                                           "15",
+                                                                           "60",
+                                                                           "1000",
+                                                                           "8000"), user_name)
+                if output.exit_code == 0:
+                    TestRun.fail("Setting cleaning policy parameters should fail!")
+            except CmdException:
+                TestRun.LOGGER.info("Non-root user cannot set alru cleaning policy parameters.")
+            output = run_as_other_user(cli.set_param_cleaning_cmd(
+                str(cache.cache_id), "acp"), user_name)
+            if output.exit_code == 0:
+                TestRun.fail("Setting cleaning policy should fail!")
+            try:
+                output = run_as_other_user(cli.set_param_cleaning_acp_cmd(str(cache.cache_id),
+                                                                          "15",
+                                                                          "1000"), user_name)
+                if output.exit_code == 0:
+                    TestRun.fail("Setting cleaning policy parameters should fail!")
+            except CmdException:
+                TestRun.LOGGER.info("Non-root user cannot set acp cleaning policy parameters.")
+        except CmdException:
+            TestRun.LOGGER.info("Non-root user cannot set cleaning policy and its parameters.")
```

Comment:
Construction like this
```
try:
   try:
      this_function_raises()
   except:
      log_exception()
except:
   second_log()
```
Will never print the second log because the innermost except will catch the exception. Can we just make it a single series of try:/excepts:? I think it will be much more readable and will serve the same purpose (well, except for those unhandled calls, but I think it makes sens to wrap each call in the same way).

Reply:
Ok, done.